packages/components/tests/dummy/app/templates/components/side-nav.hbs

```diff
@@ -0,0 +1,297 @@
+{{page-title "SideNav Component"}}
+
+<Shw::Text::H1>SideNav</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Standalone cases:</Shw::Text::H2>
+
+  <table class="shw-side-nav-table-layout">
```

Comment:
I'll replace these tables. I just wanted to get everything copied over from Cloud-ui and be sure it was working first.